src/runtime/containerd-shim-v2/delete.go

```diff
@@ -29,7 +29,7 @@ func deleteContainer(ctx context.Context, s *service, c *container) error {
 
 	// Run post-stop OCI hooks.
 	if err := katautils.PostStopHooks(ctx, *c.spec, s.sandbox.ID(), c.bundle); err != nil {
-		return err
+		shimLog.WithError(err).Warn("Failed to run post-stop hooks")
```

Comment:
I would add a comment referencing https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle here to make it clear why this is being done (as it looks like a bug not to return the error).